### DISCUSSION TITLE
Error when rendering mermaid code

### DISCUSSION THREAD

**Discussion by pyxisalpha** [quarto-dev/quarto-cli#5460] [Q&A] [answered]
I want to use mermaid to display a gantt diagram in my quarto document with the following code:
`
```{mermaid}
gantt
    title Preliminary Planning
    dateFormat DD-MM-YYYY
    section Literature review
    Literature research ARGs: L1, 05-07-2023, 30d
    Expand Thesis with findings: L2, 15-07-2023, 20d
```
`
However, when I try to render it to pdf I get the following error message:

> ERROR: NotFound: No such file or directory (os error 2)
> 
> NotFound: No such file or directory (os error 2)
>     at opRun (deno:runtime/js/40_process.js:33:16)
>     at Object.run (deno:runtime/js/40_process.js:116:17)
>     at criClient (file:///Applications/quarto/bin/quarto.js:57319:26)
>     at async file:///Applications/quarto/bin/quarto.js:57293:22
>     at async Semaphore.runExclusive (file:///Applications/quarto/bin/quarto.js:56977:29)
>     at async Object.createPngsFromHtml (file:///Applications/quarto/bin/quarto.js:57474:44)
>     at async makePng (file:///Applications/quarto/bin/quarto.js:84018:72)
>     at async makeDefault (file:///Applications/quarto/bin/quarto.js:84076:24)
>     at async Object.cell (file:///Applications/quarto/bin/quarto.js:84099:20)
>     at async Promise.all (index 0)

Does mermaid for quarto not support gantt diagrams?

Thank you for your help.

Edit:
Quarto Version: 1.3.313
OS: MacOS Ventura 13.3.1(a)
IDE: rStudio version: 2023.03.0+386
I save all the files on a github repository.

[_Planning.qmd.zip](https://github.com/quarto-dev/quarto-cli/files/11434487/_Planning.qmd.zip)

**Comment by mcanouil** [comment]
I cannot reproduce your issue.
<img width="1624" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/faebf81b-6435-42e7-af0f-12d4df5f5eed">

Could you format your post following <https://quarto.org/bug-reports.html#formatting-make-githubs-markdown-work-for-us> and provide a complete (but small) Quarto document?
Also could you provide the version of Quarto you are using?

**Reply by pyxisalpha** [reply]
I updated the original. 

Also, I am exporting it to a pdf-file using a custom .yml file. 
Hope that also helps.

Thank you for your help

Edit:
I tried running the code in Visual Studio Code as well, and it works for rendering to HTML, but gives the error shown above for PDF and DOCX

**Reply by mcanouil** [reply]
Please simply copy/paste the Quarto document using proper formatting as described in the link of my reply.

**Reply by pyxisalpha** [reply]
````
---
title: "Planning"
format: pdf
    
---
```{mermaid}
gantt
    title Preliminary Planning
    dateFormat DD-MM-YYYY
    section Literature review
    Literature research ARGs: L1, 05-07-2023, 30d
    Expand Thesis with findings: L2, 15-07-2023, 20d
```
````

**Reply by mcanouil** [reply, marked as answer]
Thanks, this is an already opened issue, see #2214.
You might need to have chrome installed on your machine.

**Reply by mcanouil** [reply]
Could you share the output of `quarto check`?

**Reply by pyxisalpha** [reply]
> Thanks, this is an already opened issue, see #2214.
> You might need to have chrome installed on your machine.

Yes, I just installed Chrome and that fixed it!

And the output:
```
[✓] Checking versions of quarto binary dependencies...
      Pandoc version 3.1.1: OK
      Dart Sass version 1.55.0: OK
[✓] Checking versions of quarto dependencies......OK
[✓] Checking Quarto installation......OK
      Version: 1.3.313
      Path: /Applications/quarto/bin

[✓] Checking basic markdown render....OK

[✓] Checking Python 3 installation....(None)

      Unable to locate an installed version of Python 3.
      Install Python 3 from https://www.python.org/downloads/

[✓] Checking R installation...........OK
      Version: 4.2.3
      Path: /Library/Frameworks/R.framework/Resources
      LibPaths:
        - /Library/Frameworks/R.framework/Versions/4.2-arm64/Resources/library
      knitr: 1.42
      rmarkdown: 2.21

[✓] Checking Knitr engine render......OK
```
